src/services/shortLink/test/index.test.ts

```diff
@@ -0,0 +1,552 @@
+        expect(response.statusCode).toEqual(StatusCodes.FORBIDDEN);
+      });
+      it('Forbidden if post short links with write permission', async () => {
+        const { item } = await mockPostData({
+          author: bob,
+          member: actor,
+          permission: PermissionLevel.Write,
+        });
+        const response = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(response.statusCode).toEqual(StatusCodes.FORBIDDEN);
+      });
+      it('Forbidden if post short links without permission on public', async () => {
+        const { item } = await mockPostData({
+          author: bob,
+          setPublic: true,
+        });
+
+        const response = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(response.statusCode).toEqual(StatusCodes.FORBIDDEN);
+      });
+    });
+
+    describe('Invalid Alias and body', () => {
+      afterEach(async () => {
+        const check = await injectGet(app, MOCK_ALIAS);
+        expect(check.statusCode).toEqual(StatusCodes.NOT_FOUND);
+      });
+
+      it('Bad request if post short links with alias < 6 chars', async () => {
+        const { item } = await mockPostData({ author: actor });
+
+        const response = await injectPost(
+          app,
+          mockShortLinkPayload({ item_id: item.id, alias: '12345' }),
+        );
+        expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST);
+      });
+      it('Bad request if post short links with invalid alias chars', async () => {
+        const { item } = await mockPostData({ author: actor });
+
+        const response = await injectPost(
+          app,
+          mockShortLinkPayload({ item_id: item.id, alias: '1_2$3<45' }),
+        );
+        expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST);
+      });
+
+      it('Bad request if post short links without body', async () => {
+        const response = await injectPost(app);
+        expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST);
+      });
+
+      it('Bad request if post short links with empty body', async () => {
+        const response = await injectPost(app, {});
+        expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST);
+      });
+    });
+
+    describe('Not Found', () => {
+      it('Conflict if post short links with already exist alias', async () => {
+        const { item } = await mockPostData({ author: actor });
+        const { item: item2 } = await mockPostData({ author: actor });
+
+        await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+
+        const response = await injectPost(app, mockShortLinkPayload({ item_id: item2.id }));
+        expectException(response, new ShortLinkDuplication(MOCK_ALIAS));
+      });
+    });
+
+    describe('Not Found', () => {
+      it('Not found if post short links with invalid item id', async () => {
+        const response = await injectPost(app, mockShortLinkPayload());
```

Comment:
Here too, maybe specify the itemId you want to use, and add a comment that the item does not exist prior.